src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -156,17 +156,23 @@ public void initHoneybadger()
         if(environment == null)
             environment = new SulEnvironment();
 
-        String apiKey = environment.getEnvironmentVariable(HONEYBADGER_KEY);
+        String apiKey = environment.getEnvironmentVariable(HONEYBADGER_API_KEY_ENV_VAR);
+        String honeybadgerEnv = environment.getEnvironmentVariable(HONEYBADGER_ENV_NAME_ENV_VAR);
```

Comment:
well... normally i'd be in favor of making stuff constant where possible... but in this case, i think we can't do that at class declaration time, because the values come from `environment`, which doesn't get initialized until object instantiation.